src/device/device-management.ts

```diff
@@ -43,7 +43,10 @@ export type VideoDeviceConstraints = {
 };
 
 /**
- * Creates a camera video track.
+ * Creates a camera video track. Please note that the constraint params in second getUserMedia call would NOT take effect when:
+ *
+ * 1. Previous captured video track is not stopped.
```

Comment:
should this say "previous captured video track from the same device"?

Reply:
done